wordpress/1.9/README.md

```diff
@@ -0,0 +1,149 @@
+# How to use WordPress on DC/OS
+
+[WordPress](https://wordpress.com/) is a popular blogging platform.
+
+- Estimated time for completion: 10 minutes
+- Target audience: Anyone interested using a blog.
+- Scope: Learn how to use WordPress on DC/OS.
+
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Marathon-LB](#install-marathon-lb)
+- [Install MySQL](#install-mysql)
+- [Install WordPress](#install-wordpress)
+- [Access WordPress](#access-wordpress)
+- [Use WordPress in production](#use-wordpress-in-production)
+
+## Prerequisites
+
+- A running DC/OS 1.9 cluster with at least 1 [public agent](https://dcos.io/docs/1.9/overview/concepts/#public) node with 2 CPUs and 1 GB of RAM available.
```

Comment:
The address has changed to https://dcos.io/docs/1.9/overview/concepts/#public-agent-node for 1.9